packages/types/src/electra/sszTypes.ts

```diff
@@ -14,8 +25,65 @@ import {ssz as bellatrixSsz} from "../bellatrix/index.js";
 import {ssz as capellaSsz} from "../capella/index.js";
 import {ssz as denebSsz} from "../deneb/index.js";
 
-const {UintNum64, Slot, Root, BLSSignature, UintBn256, Bytes32, BLSPubkey, DepositIndex, UintBn64, ExecutionAddress} =
-  primitiveSsz;
+const {
+  UintNum64,
+  Slot,
+  Root,
+  BLSSignature,
+  UintBn256,
+  Bytes32,
+  BLSPubkey,
+  DepositIndex,
+  UintBn64,
+  ExecutionAddress,
+  ValidatorIndex,
+} = primitiveSsz;
+
+export const AggregationBits = new BitListType(MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT);
+
+export const CommitteeBits = new BitVectorType(MAX_COMMITTEES_PER_SLOT);
+
+export const AttestingIndices = new ListBasicType(
+  ValidatorIndex,
+  MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT
+);
+
+export const Attestation = new ContainerType(
+  {
+    aggregationBits: AggregationBits,
+    data: phase0Ssz.AttestationData,
+    committeeBits: CommitteeBits,
```

Comment:
Maybe add a comment here signalling to devs reading the code that this is the addition

Reply:
yes this `CommitteeBits` is a confusing part between phase0 vs electra. I suggest:
- refactor `phase0.CommitteeBits` to be `phase0.ParticipationBits`
- so that we keep this `electra.CommitteeBits` align to the spec

Reply:
> > refactor phase0.CommitteeBits to be phase0.ParticipationBits
> 
> this could be a `TODO` to speed up electra

Added thanks